### PR TITLE
❄️ Disabled multiple flaky visual diff tests

### PR DESCRIPTION
**Pull request by danielrozenberg**
And reassign all TODOs with issue number and working-groups.

Related to #27181

### CODE REVIEW

File: test/visual-diff/visual-tests [hunk 1]
```diff
@@ -220,13 +222,17 @@
       "name": "amp-sidebar toolbar > ol",
       "viewport": {"width": 400, "height": 800},
       "interactive_tests": "examples/visual-tests/amp-sidebar/amp-sidebar-toolbar.js",
-    },{
+    },
+    {
+      // TODO(#27456, @ampproject/wg-ui-and-a11y): see https://percy.io/ampproject/amphtml/builds/4637948
```
Comment (rsimha) [inline test/visual-diff/visual-tests]: Is this missing a `"flaky": true`?

File: test/visual-diff/visual-tests [hunk 2]
```diff
@@ -690,7 +698,7 @@
     },
     {
       "flaky": true,
-      // TODO(#22162, cvializ): see https://percy.io/ampproject/amphtml/builds/1828636
+      // TODO(#22162, @ampproject/wg-ui-and-a11y): see https://percy.io/ampproject/amphtml/builds/1828636
```
Comment (rsimha) [inline test/visual-diff/visual-tests]: Same here. Missing `"flaky": true`?

**Comment by rsimha** [comment]
After these skips, I still see a couple of failing visual tests: https://percy.io/ampproject/amphtml/builds/4690468

New flakes? Missing `"flaky": true` directives?

File: test/visual-diff/visual-tests (hunk 2 above)
Comment (alanorozco) [reply test/visual-diff/visual-tests]: See line above comment :) 

(Fixing in #27461.)

File: test/visual-diff/visual-tests (hunk 1 above)
Comment (danielrozenberg) [reply test/visual-diff/visual-tests]: Correct. Fixed.

File: test/visual-diff/visual-tests (hunk 2 above)
Comment (danielrozenberg) [reply test/visual-diff/visual-tests]: It was above the comment, moved it below for consistency

Comment (rsimha) [reply test/visual-diff/visual-tests]: I'm going to blame Github for hiding the previous line. Also, today is Friday 😃

**Comment by danielrozenberg** [comment]
> After these skips, I still see a couple of failing visual tests: https://percy.io/ampproject/amphtml/builds/4690468
> 
> New flakes? Missing `"flaky": true` directives?

These are entirely new tests. Added flaky tag to them :)

File: test/visual-diff/visual-tests [hunk 3]
```diff
@@ -323,6 +330,8 @@
       ]
     },
     {
+      // TODO(#27452, @ampproject/wg-stories): font size is unstable, e.g., https://percy.io/ampproject/amphtml/builds/4680570
```
Comment (gmajoulet) [inline test/visual-diff/visual-tests]:
```suggestion
      // TODO(#27185, @ampproject/wg-stories): font size is unstable, e.g., https://percy.io/ampproject/amphtml/builds/4680570
```

File: test/visual-diff/visual-tests [hunk 4]
```diff
@@ -456,6 +465,8 @@
       "interactive_tests": "examples/visual-tests/amp-story/amp-story-consent.js"
     },
     {
+      // TODO(#27452, @ampproject/wg-stories): font size is unstable, e.g., https://percy.io/ampproject/amphtml/builds/4680570
```
Comment (gmajoulet) [inline test/visual-diff/visual-tests]:
```suggestion
      // TODO(#27185, @ampproject/wg-stories): font size is unstable, e.g., https://percy.io/ampproject/amphtml/builds/4680570
```

**Pull request merged**